science-containers/Dockerfiles/firefly/Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM gradle:jdk21 AS builder
+FROM gradle:jdk21 AS base
 
 COPY . /firefly
 WORKDIR /firefly/cadc-sso
 
 SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
 
+FROM base AS builder
 RUN set -eux; \
     gradle build --info \
     && ls -l lib/build/libs/cadc-sso-*.jar
 
-FROM ipac/firefly:2024.3.5
-COPY --from=builder /firefly/cadc-sso/lib/build/libs/cadc-sso-*.jar /usr/local/tomcat/extensions/
-COPY --from=builder /firefly/launchTomcat.sh /usr/local/tomcat/
-
+# Currently, the CADC-SSO Adapter only works with the nightly build of Firefly
+FROM ipac/firefly:nightly
+COPY --from=builder /firefly/entrypoint.py /usr/local/tomcat/
```

Comment:
The `/firefly/entrypoint.py` doesn't exist in the `builder` base image.  Is this leftover?